# Brazilian Portuguese translation for openobject-addons
# Copyright (c) 2012 Rosetta Contributors and Canonical Ltd 2012
# This file is distributed under the same license as the openobject-addons package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2012.
#
msgid ""
msgstr ""
"Project-Id-Version: openobject-addons\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2012-02-08 00:36+0000\n"
"PO-Revision-Date: 2012-07-28 23:01+0000\n"
"Last-Translator: Fábio Martinelli <webmaster@guaru.net>\n"
"Language-Team: Brazilian Portuguese <pt_BR@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2012-07-30 05:10+0000\n"
"X-Generator: Launchpad (build 15702)\n"

#. module: google_base_account
#: field:res.users,gmail_user:0
msgid "Username"
msgstr "Usuário"

#. module: google_base_account
#: model:ir.actions.act_window,name:google_base_account.act_google_login_form
msgid "Google Login"
msgstr "Login do Google"

#. module: google_base_account
#: code:addons/google_base_account/wizard/google_login.py:29
#, python-format
msgid "Google Contacts Import Error!"
msgstr "Erro de importação dos contatos Google!"

#. module: google_base_account
#: view:res.users:0
msgid " Synchronization "
msgstr " Sincronização "

#. module: google_base_account
#: sql_constraint:res.users:0
msgid "You can not have two users with the same login !"
msgstr "Você não pode ter dois usuários com o mesmo login!"

#. module: google_base_account
#: code:addons/google_base_account/wizard/google_login.py:75
#, python-format
msgid "Error"
msgstr "Erro"

#. module: google_base_account
#: view:google.login:0
msgid "Google login"
msgstr "Login do Google"

#. module: google_base_account
#: model:ir.model,name:google_base_account.model_res_users
msgid "res.users"
msgstr "res.users"

#. module: google_base_account
#: field:google.login,password:0
msgid "Google Password"
msgstr "Senha do Google"

#. module: google_base_account
#: view:google.login:0
msgid "_Cancel"
msgstr "_Cancelar"

#. module: google_base_account
#: view:res.users:0
msgid "Google Account"
msgstr "Conta do Google"

#. module: google_base_account
#: field:google.login,user:0
msgid "Google Username"
msgstr "Usuário do Google"

#. module: google_base_account
#: code:addons/google_base_account/wizard/google_login.py:29
#, python-format
msgid ""
"Please install gdata-python-client from http://code.google.com/p/gdata-"
"python-client/downloads/list"
msgstr ""
"Por favor instale gdata-python-client de http://code.google.com/p/gdata-"
"python-client/downloads/list"

#. module: google_base_account
#: model:ir.model,name:google_base_account.model_google_login
msgid "Google Contact"
msgstr "Contatos do Google"

#. module: google_base_account
#: view:google.login:0
msgid "_Login"
msgstr "_Login"

#. module: google_base_account
#: constraint:res.users:0
msgid "The chosen company is not in the allowed companies for this user"
msgstr ""
"A empresa escolhida não está entre as empresas habilitadas para este usuário"

#. module: google_base_account
#: field:res.users,gmail_password:0
msgid "Password"
msgstr "Senha"

#. module: google_base_account
#: code:addons/google_base_account/wizard/google_login.py:75
#, python-format
msgid "Authentication fail check the user and password !"
msgstr "Usuário e senha não conferem!"

#. module: google_base_account
#: view:google.login:0
msgid "ex: user@gmail.com"
msgstr "ex: usuario@gmail.com"
